mllib-tests/v1p5/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -237,11 +237,17 @@ class GLMClassificationTest(sc: SparkContext) extends GLMTests(sc) {
     }
 
     if (regType == "elastic-net") {  // use spark.ml
+      val solver = optimizer match {
+        case "l-bfgs" =>
+
```

Comment:
`solver` is not used. It'd be better to say `assert(optimizer == "l-bfgs", "...")`.